src/input/InputSDL.cpp

```diff
@@ -64,6 +64,9 @@ void InputSDLWorker::run()
 {
   QElapsedTimer polltimer;
 
+  QMap<int,int> repeats;
```

Comment:
I know we are inconsistent with this in other places in the code, but we should use `QHash` instead of `QMap`. Map is only useful when we need something to be sorted.